Assignment2/u120210.rdf

```diff
@@ -0,0 +1,9 @@
+@base <http://www.oeg-upm.net/ontologies/people/ >
+@prefix vocab: <http://www.ontologies.org/ontologies/ejercicio2>
+:Class01 vocab:includes :Sensor029 ;
+vocab:includes: Computer101.
+:Sensor029 vocab:hasMeasurement :Measurement8401.
+:Computer01 vocab:hasOwner :User10A.
+:Measurement8401 vocab:hasTemperature "29" ;
+vocab:atTime "2010-06-12T12:00:12".
```

Comment:
Same here with the typing of the date (and check whether it is compatible with the format required by the corresponding XSD datatype that you select)